packages/design-system/src/components/N8nNodeIcon/NodeIcon.vue

```diff
@@ -30,7 +30,6 @@
 				</div>
 				<div v-else :class="$style.nodeIconPlaceholder">
 					{{ nodeTypeName ? nodeTypeName.charAt(0) : '?' }}
-					?
```

Comment:
do we need to do this for the tooltip on line 20 as well?

Reply:
Oh good catch! Removed it from there as well